sbthost/nsc/src/main/scala/scala/meta/internal/sbthost/SbthostPipeline.scala

```diff
@@ -40,30 +40,32 @@ trait SbthostPipeline extends DatabaseOps { self: SbthostPlugin =>
           //          global.reporter.warning(NoPosition, s"Unknown reporter $els")
           mutable.LinkedHashSet.empty
       }
-    def getNames(unit: g.CompilationUnit): Seq[s.ResolvedName] = {
-      val buffer = ListBuffer.newBuilder[s.ResolvedName]
-      object traverser extends g.Traverser {
-
-        def isValidSymbol(symbol: g.Symbol) =
-          symbol.ne(null) && symbol != g.NoSymbol
+    override def newPhase(prev: Phase) = new StdPhase(prev) {
+      def apply(unit: g.CompilationUnit): Unit = {
+        val names = ListBuffer.newBuilder[s.ResolvedName]
+        val denots = ListBuffer.newBuilder[s.SymbolDenotation]
+        def getNames(): Unit = {
+          object traverser extends g.Traverser {
+            def isValidSymbol(symbol: g.Symbol) =
+              symbol.ne(null) && symbol != g.NoSymbol
+            override def traverse(tree: g.Tree): Unit = {
+              if (tree.pos.isDefined &&
+                  tree.hasSymbol &&
+                  isValidSymbol(tree.symbol) &&
+                  isValidSymbol(tree.symbol.owner)) {
```

Comment:
What are we filtering here concretely? I guess it's syntactic members generated by scalac